godotcord_user.h

```diff
@@ -0,0 +1,17 @@
+#ifndef GODOTCORD_USER_H
+#define GODOTCORD_USER_H
+
+#include "core/reference.h"
+
+class GodotcordUser : public Reference {
+	GDCLASS(GodotcordUser, Reference)
+public:
+	uint64_t id = -1;
```

Comment:
You need to expose the attributes to GDScript like you did in the other classes (using GET_SET_COMBO and GODOTCORD_PROPERTIES)